composer.json

```diff
@@ -16,13 +16,13 @@
     }
   ],
   "require": {
-    "php": "^7.4 || ^8.0",
-    "doctrine/annotations": "^1.0.0",
-    "laminas/laminas-code": "^4.0.0",
+    "php": ">=7.4",
```

Comment:
also, I was wondering if I should push this to >=8.0 in order to keep roave/better-reflection @ 5.0 then I will be able to remove [this check ](https://github.com/researchgate/injektor/blob/master/test/rg/injektor/FactoryOnlyDependencyInjectionContainerTest.php#L55)in the test cases.

Reply:
I think in this special case it's good to require PHP 8.0 to be able to remove that extra code.
Because the previous version was supporting both 7.4 and 8.0 (unlike `roave/better-reflection` 😞 ) it's easy enough to go from 7.4 to 8.0 and now from 8.0 to beyond.